srcs/incs/WebServer.hpp

```diff
@@ -6,18 +6,22 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <string>
-#include <vector>
+#include <map>
 #include <Config.hpp>
 #include <Server.hpp>
 
 class WebServer {
 	private:
-		Config	config_;
-		std::vector<Server>	servers_;
-		fd_set	master_set_;
-		fd_set	read_set_;
-		fd_set	write_set_;
-		int		max_sd_;
+		typedef	int							Socket;
+		typedef std::map<Socket, Server>	ServersMap;
```

Comment:
Good idea move from vector to map 👌Also wanted to tell you, love your approach of using typedefs, makes code far more readable!